### ISSUE TITLE
Flycheck doesn't find tslint in local npm package

### ISSUE THREAD

**Issue by nickmccurdy** [flycheck/flycheck#1391]
I have an npm package that is using both eslint and tslint. Both are installed locally in `node_modules`. `flycheck-verify-setup` recognizes my locally installed eslint, but it says that tslint is not installed (even though it's in the same directory) unless I install it globally.

```
Flycheck version: 32snapshot (package: 20180123.1419)
Emacs version:    25.3.1
System:           x86_64-apple-darwin13.4.0
Window system:    ns
```

**Comment by fmdkdd** [comment]
Looks like a `PATH` issue?  Both checkers should be using `flycheck-executable-find` to locate the executables.  So, for Flycheck to find the locally installed `eslint`, you probably customized that variable?

**Comment by nickmccurdy** [comment]
Sorry, false alarm. I use `add-node-modules-path` to build my exec path and I forgot to add a hook for `typescript-mode`, now flycheck finds my local tslint as expected.
